contrib/tag-release.py

```diff
@@ -47,15 +31,9 @@ def main():
 
     solana_version = f'{solana_version_major}{solana_version_minor:02d}{solana_version_patch:02d}'
 
-    with open('src/app/fdctl/version.mk', 'w') as f:
```

Comment:
You need to write back `<minor>.<patch>` to the version.txt as part of this, so tag-release keeps incrementing it.